README.md

```diff
@@ -106,14 +106,24 @@ class MyCustomAppIntro : AppIntro() {
 
 Please note that you **must NOT call** setContentView. The `AppIntro` superclass is taking care of it for you.
 
+Next, define your app intro theme in res/values/styles.xml.
+```
+  <style name="AppIntroTheme" parent="Theme.AppCompat.NoActionBar">
+        <item name="colorPrimary">#F57C00</item>
+        <item name="colorPrimaryDark">#212121</item>
+        <item name="colorAccent">#F57C00</item>
+    </style>
+```
```

Comment:
I won't add this into the README honestly. While I understand this fixed your issue, this is not a required step for all of our users.